pkg/api/v1/types.go

```diff
@@ -2633,6 +2655,9 @@ type ResourceQuotaSpec struct {
 	// Hard is the set of desired hard limits for each named resource.
 	// More info: http://releases.k8s.io/HEAD/docs/design/admission_control_resource_quota.md#admissioncontrol-plugin-resourcequota
 	Hard ResourceList `json:"hard,omitempty"`
+	// Scopes define filters that must match each object tracked by a quota.
+	// If not specified, the quota matches all objects.
+	Scopes []ResourceQuotaScope `json:"scopes,omitempty"`
```

Comment:
As mentioned in the proposal doc:

Comments shouldn't mention Go fields/types

I guess this doesn't really need patchStrategy